exec-analysis/src/main/java/com/capitalone/dashboard/exec/model/HygieiaSparkQuery.java

```diff
@@ -61,9 +75,15 @@
                     "cast(from_unixtime(cast(closedTime/1000 as bigint)) as timestamp) as closeDate, closedTime, " +
                     "cast(count(*) as double) as count " +
                     "FROM incident " +
-                    "WHERE LTRIM(RTRIM(IFNULL(affectedItem, ''))) != '' and severity in ('1','2','3','3C','3D') " +
-                    " and LTRIM(RTRIM(IFNULL(status, ''))) != '' and UCASE(status) in ('OPEN', 'OPENED', 'CLOSE', 'CLOSED')" +
+                    "WHERE LTRIM(RTRIM(IFNULL(affectedItem, ''))) != '' and severity in ('P1','P2','P3','P4') " +
```

Comment:
What is the change for? If this needs to be changed then it should be a property based configuration.

Reply:
The severity P1, P2 and so on will use internally in T-Mobile, reverted to original state.